ibis/expr/datatypes/value.py

```diff
@@ -150,6 +150,12 @@ def infer_enum(_: enum.Enum) -> dt.String:
     return dt.string
 
 
+@infer.register(decimal.Decimal)
+def infer_decimal(value: decimal.Decimal) -> dt.Decimal:
+    """Infer the [`Decimal`][ibis.expr.datatypes.Decimal] type of `value`."""
```

Comment:
No need for a docstring here, lets remove it to be consistent with the other dispatches.

```suggestion
```

Reply:
@jcrist Thanks for the speedy review! I've added the test. I'm also happy to do this, but wanted to point out that a lot of the dispatches have this docstring (from where I copied it), while others don't. I can standardize it across the dispatches, if you feel strongly one way or another, or I can just delete the docstring from this one.

There's also other minor inconsistencies, like some of the docstrings saying `value` even though the parameter name for them is `values`.

Reply:
Ah, you're right. No need to drop the docstring here, and don't worry about standardizing things. Lets leave it as is.